<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi Movie in a Dir Finder</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../SABUtils/resources/SABUtils.qrc">
    <normaloff>:/SABUtilsResources/open.png</normaloff>:/SABUtilsResources/open.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget_1">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QToolButton" name="btnSelectLHSFile">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../SABUtils/resources/SABUtils.qrc">
             <normaloff>:/SABUtilsResources/open.png</normaloff>:/SABUtilsResources/open.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QTableWidget" name="lhsData">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="lhsFile">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLineEdit" name="rhsFile"/>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="btnSelectRHSFile">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../SABUtils/resources/SABUtils.qrc">
             <normaloff>:/SABUtilsResources/open.png</normaloff>:/SABUtilsResources/open.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QTableWidget" name="rhsData">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="saveBtn">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="compareBtn">
           <property name="text">
            <string>Compare</string>
           </property>
           <property name="icon">
            <iconset resource="../SABUtils/resources/SABUtils.qrc">
             <normaloff>:/SABUtilsResources/run.png</normaloff>:/SABUtilsResources/run.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QTableView" name="mergeData">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Results</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <widget class="QStackedWidget" name="resultsPages">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>4</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="summary">
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QGroupBox" name="groupBox_5">
                <property name="title">
                 <string>Summary</string>
                </property>
                <layout class="QFormLayout" name="formLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>LHS Rows:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="numLHSRows">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>RHS Rows:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="numRHSRows">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Number of Match Columns</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLineEdit" name="numMatchedColumns">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Number of LHS Only Rows:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QLineEdit" name="numLHSOnly">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Number of RHS Only Rows:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QLineEdit" name="numRHSOnly">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Number of Matched Rows:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="1">
                  <widget class="QLineEdit" name="numMatchedRows">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="0">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Merged Row Count:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="1">
                  <widget class="QLineEdit" name="numTotalRows">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="IgnoredRows">
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QGroupBox" name="groupBox_6">
                <property name="title">
                 <string>Ignored Rows</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_10">
                 <item>
                  <widget class="QSplitter" name="splitter_6">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <widget class="QListWidget" name="ignoredRowsLHS">
                    <property name="alternatingRowColors">
                     <bool>true</bool>
                    </property>
                   </widget>
                   <widget class="QListWidget" name="ignoredRowsRHS">
                    <property name="alternatingRowColors">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="mergedColumns">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>Merged Colums:</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QSplitter" name="splitter_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <widget class="QTreeWidget" name="mergedColumnsLHS">
                    <property name="alternatingRowColors">
                     <bool>true</bool>
                    </property>
                    <column>
                     <property name="text">
                      <string>Original</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>New</string>
                     </property>
                    </column>
                   </widget>
                   <widget class="QTreeWidget" name="mergedColumnsRHS">
                    <property name="alternatingRowColors">
                     <bool>true</bool>
                    </property>
                    <column>
                     <property name="text">
                      <string>Original</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>New</string>
                     </property>
                    </column>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="extraColumns">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>Extra Columns</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QSplitter" name="splitter_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <widget class="QListWidget" name="extraColumnsLHS">
                    <property name="alternatingRowColors">
                     <bool>true</bool>
                    </property>
                   </widget>
                   <widget class="QListWidget" name="extraColumnsRHS">
                    <property name="alternatingRowColors">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="matchedColumns">
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QGroupBox" name="groupBox_4">
                <property name="title">
                 <string>Matched Columns</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QSplitter" name="splitter_4">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <widget class="QListWidget" name="matchedColumnsLHS">
                    <property name="alternatingRowColors">
                     <bool>true</bool>
                    </property>
                   </widget>
                   <widget class="QListWidget" name="matchedColumnsRHS">
                    <property name="alternatingRowColors">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QTreeWidget" name="resultsTree">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="headerHidden">
             <bool>true</bool>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="headerShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Items</string>
             </property>
            </column>
            <item>
             <property name="text">
              <string>Summary</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ignored Rows</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Merged Columns</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Extra Columns</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Matched Columns</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1035</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Merged Data...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lhsFile</tabstop>
 </tabstops>
 <resources>
  <include location="../SABUtils/resources/SABUtils.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>CMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
